sdks/python/apache_beam/transforms/trigger.py

```diff
@@ -378,6 +378,8 @@ class AfterCount(TriggerFn):
   COUNT_TAG = _CombiningValueStateTag('count', combiners.CountCombineFn())
 
   def __init__(self, count):
+    if count == 0:
```

Comment:
<!--new_thread; commit:17604fb4ecfba607daa3c22312c02865b02ecee7; resolved:0-->
This is checking for non-zero numbers instead of the positive number. You can change this to `count > 0` to check for positive numbers.